tailcfg/tailcfg.go

```diff
@@ -1797,8 +1798,26 @@ type MapResponse struct {
 	// Note that this package's type, due its use of a slice and omitempty, is
 	// unable to marshal a zero-length non-nil slice. The control server needs
 	// to marshal this type using a separate type. See MapResponse docs.
+	//
+	// See PacketFilters for the newer way to send PacketFilter updates.
 	PacketFilter []FilterRule `json:",omitempty"`
 
+	// PacketFilters is a method to send incremental packet filter updates to
+	// the client without having to send the entire packet filter on any
+	// changes. The map keys are server-assigned arbitrary strings. The map
+	// values are the new rules for that key, or nil to delete it. The client
+	// then concatenates all the rules together to generate the final packet
+	// filter. Because the FilterRules can only match or not match, the ordering
+	// of filter rules doesn't matter.
+	//
+	// If the server sends a non-nil PacketFilter (above), that is equivalent to
+	// a named packet filter with the key "base".
+	//
+	// As a special case, the map key "*" with a value of nil means to clear all
+	// prior named packet filters (including any implicit "base") before
+	// processing the other map entries.
+	PacketFilters map[string][]FilterRule `json:",omitempty"`
```

Comment:
NamedPacketFilters?

Reply:
I think we'd regret that name in a year. It'll sound like "NewWayPacketFilters"?

Reply:
off by one fields 🤮 , so easy to typo `PacketFilter` instead of `PacketFilters` but at least it's a different type